src/FilenameParsing/NaturalFileIDHelper.php

```diff
@@ -23,10 +21,16 @@ public function parseFileID($fileID)
         }
 
         $filename = $matches['folder'] . $matches['basename'] . $matches['extension'];
+        $variant = isset($matches['variant']) ? $matches['variant'] : '';
+
+        if (isset($variant)) {
```

Comment:
```suggestion
        $variant = $matches['variant'] ?: '';

        if ($variant) {
```

Reply:
This isn't new code but I'll validate and make a change here to avoid PR ping pong

Reply:
Done